core/api.py

```diff
@@ -0,0 +1,66 @@
+# This file defines the functions used to request data from OIT's APIs
+
+import requests
+from datetime import datetime, timedelta
```

Comment:
I prefer to keep multiple imports from the same module on separate lines, since it makes git history more readable when things are added/removed, so this would be...

```
from datetime import datetime
from datetime import timedelta
```